lib/ansible/modules/cloud/amazon/dms_replication_subnet_group.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: dms_replication_subnet_group
+short_description: creates or destroys a data migration services subnet group
+description:
+    - creates or destroys a data migration services subnet group
+version_added: "2.9"
+options:
+    state:
+        description:
+            - State of the subnet group
+        default: present
+        choices: ['present', 'absent']
```

Comment:
not a blocker but please add `type: <...>` for all params, it will be sanity checked in the future.